pymc/tests/test_distributions_moments.py

```diff
@@ -127,6 +128,21 @@ def test_halfnormal_moment(sigma, size, expected):
     assert_moment_is_expected(model, expected)
 
 
+@pytest.mark.parametrize(
+    "nu, sigma, size, expected",
+    [
+        (1, 1, None, 1),
+        (1, 1, 5, np.ones(5)),
+        (1, np.arange(5), None, np.arange(5)),
```

Comment:
You should add a test where nu is a vector and sigma is a scalar. That would have caught the issue I mentioned above. 

Something like this:
```suggestion
        (1, np.arange(5), None, np.arange(5)),
        (np.arange(5), 1, None, np.full(5, 1)),
```

Reply:
Thanks for the review @ricardoV94, added a new commit with both suggestions.